mackerel-plugin-docker/lib/docker.go

```diff
@@ -525,6 +539,19 @@ func Do() {
 		docker.pathBuilder = pb
 	}
 
+	if *optCpuFormat == "percentage" {
+		if docker.Method == "File" {
+			log.Fatalf("'--cpu-format percentage' is not supported with File method.")
+		}
+		docker.UseCpuPercentage = true
+	} else if *optCpuFormat == "usage" {
+		docker.UseCpuPercentage = false
+	} else {
+		if docker.Method == "API" {
+			docker.UseCpuPercentage = true
+		}
+	}
+
```

Comment:
The code seems to be same as the following code, but I don't care whichever, current code is acceptable.
```golang
if *optCpuFormat == "percentage" && docker.Method == "File" {
	log.Fatalf("'--cpu-format percentage' is not supported with File method.")
}
docker.UseCpuPercentage = *optCpuFormat == "percentage"
		|| docker.Method == "API" && *optCpuFormat == ""
```